app/javascript/src/modules/organisations.js

```diff
@@ -22,63 +18,59 @@ async function searchResults (host, searchTerm) {
     }
   })
   const data = response.data.data
-  return updateMatchCounters(data, searchTerm)
+  return data
```

Comment:
could shortcut this
```
return response.data.data
```

Reply:
assuming the const is being used elsewhere, 👍

Reply:
done